gwctl/cmd/get.go

```diff
@@ -110,12 +125,12 @@ func runGet(cmd *cobra.Command, args []string, params *utils.CmdParams) {
 		if len(args) > 1 {
 			filter.Name = args[1]
```

Comment:
We won't have any problem with this because whenever this situation occurs, it would lead the `discoverer.DiscoverResourcesForX(filter)` to generate a resourceModel with a list of 1 Node. 

This list of 1 Node/item gets fed to the `renderPrintablePayload` and that function is smart enough to render a list when len of the list > 1 and render the specific item when the len of list == 1